utils/compiler/src/PhpScoper/StaticEasyPrefixer.php

```diff
@@ -38,10 +38,6 @@ final class StaticEasyPrefixer
 
         // this is public API of a Rector rule
         'Symplify\RuleDocGenerator\*',
-
-        // not needed since Rector 0.12.7, but keep for BC layer for couple of months
-        // for configuring sets with ValueObjectInliner
-        'Symplify\SymfonyPhpConfig\*',
```

Comment:
Unfortunatelly, this must be removed again, as it breaks Symplify config downgrade:
https://github.com/symplify/symplify/runs/4484079469?check_suite_focus=true